training/datastore/utils.py

```diff
@@ -194,12 +196,14 @@ def get_query_params(self, repo_url):
         """Give the Query Parameters which are organization and package name respectively."""
         org = str()
         package_name = str()
-        url_chunks = (repo_url.rsplit('/', 2))
-        if 'github' not in url_chunks[1]:
-            org = url_chunks[1]
-        package_name = url_chunks[2]
-
-        return list([org, package_name])
+        try:
+            url_chunks = (repo_url.rsplit('/', 2))
+            if 'github' not in url_chunks[1]:
+                org = url_chunks[1]
+            package_name = url_chunks[2]
+            return list([org, package_name])
+        except Exception:
+            return list([org, package_name])
```

Comment:
I don't think it'll be possible to return this if there is an exception.

Reply:
No, it will return. I checked it :)

Reply:
Then what exception are you looking out for?

Reply:
That's the last line of the try block as well, and the only line of the catch block.